Bot/cogs/remindme.py

```diff
@@ -70,11 +70,11 @@ def cog_unload(self):
         if time_set < time_now:
             delta_time += timedelta(days=1)
             
-        await self.remindme_base(ctx,str(timedelta(seconds=int(delta_time.total_seconds()))),*,message)
+        await self.remindme_base(ctx,str(timedelta(seconds=int(delta_time.total_seconds()))),message)
         
     @commands.command(hidden=True,pass_context=True)
     async def remindme(self,ctx,get_time,*,message=""):
-        await self.remindme_base(ctx,get_time,*,message)
+        await self.remindme_base(ctx,get_time,message)
```

Comment:
This is what runtime error will give out.
`TypeError: remindme_base() takes 3 positional arguments but 4 were given`